tests/gql_behave/tests/memgraph_V1/features/hops_limit.feature

```diff
@@ -0,0 +1,255 @@
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'C'})-[:CONNECTED]->(:Node {name: 'G'})> |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'C'})-[:CONNECTED]->(:Node {name: 'F'})> |
+
+
+    Scenario: Test simple expand
+        Given graph "simple_binary_tree"
+        When executing query:
+            """
+            USING HOPS LIMIT 5 MATCH p=(a)-[:CONNECTED]->(b) RETURN p
+            """
+        Then the result should be:
+            | p                                                       |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'B'})> |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'C'})> |
+            | <(:Node {name: 'B'})-[:CONNECTED]->(:Node {name: 'D'})> |
+            | <(:Node {name: 'B'})-[:CONNECTED]->(:Node {name: 'E'})> |
+            | <(:Node {name: 'C'})-[:CONNECTED]->(:Node {name: 'F'})> |
+
+
+    Scenario: Test simple expand both directions
+        Given graph "simple_binary_tree"
+        When executing query:
+            """
+            USING HOPS LIMIT 5 MATCH p=(a)-[:CONNECTED]-(b) RETURN p
+            """
+        Then the result should be:
+            | p                                                       |
+            | <(:Node {name: 'B'})<-[:CONNECTED]-(:Node {name: 'A'})> |
+            | <(:Node {name: 'C'})<-[:CONNECTED]-(:Node {name: 'A'})> |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'B'})> |
+            | <(:Node {name: 'D'})<-[:CONNECTED]-(:Node {name: 'B'})> |
+            | <(:Node {name: 'E'})<-[:CONNECTED]-(:Node {name: 'B'})> |
+
+
+    Scenario: Test hops limit flag with partial results disabled
+        Given an empty graph
+        And having executed:
+            """
+            CREATE (a)-[:CONNECTED]->(b)
+            CREATE (a)-[:CONNECTED]->(c)
+            CREATE (a)-[:CONNECTED]->(d)
+            CREATE (a)-[:CONNECTED]->(e)
+            CREATE (a)-[:CONNECTED]->(f)
+            """
+        And having executed:
+            """
+            SET DATABASE SETTING 'hops_limit_partial_results' TO 'false'
+            """
+        When executing query:
+            """
+            USING HOPS LIMIT 2 MATCH p=(a)-[:CONNECTED *]->(e) SET e:Test RETURN p
+            """
+        Then an error should be raised
+
+
+    Scenario: Test hops limit flag with partial results enabled
+        Given an empty graph
+        And having executed:
+            """
+            CREATE (a)-[:CONNECTED]->(b)
+            CREATE (a)-[:CONNECTED]->(c)
+            CREATE (a)-[:CONNECTED]->(d)
+            CREATE (a)-[:CONNECTED]->(e)
+            CREATE (a)-[:CONNECTED]->(f)
+            """
+        When executing query:
+            """
+            USING HOPS LIMIT 5 MATCH p=(a)-[:CONNECTED *]->(e) RETURN p
+            """
+        Then the result should be:
+            | p                     |
+            | <()-[:CONNECTED]->()> |
```

Comment:
Why the nodes are empty?

Reply:
i agree, it's better to name them with labels so we can fix the ordering